index.html

```diff
@@ -0,0 +1,563 @@
+                                    </svg>
+
+                            </li>
+
+                                    <li>
+                                        <img src="./Images/Shop.png" alt="">
+                                    </li>
+                            </div>
+                          
+                                    
+                                <div class="nav__sub">
+                                    <li>
+                                            <svg width="24" height="14" viewBox="0 0 24 14" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                                                        <path
+                                                                            d="M0.571472 0H23.4286V2.28571H0.571472V0ZM6.28576 5.71429H23.4286V8H6.28576V5.71429ZM13.4286 11.4286H23.4286V13.7143H13.4286V11.4286Z"
+                                                                            fill="#737373" />
+                                                                    </svg>
+                                            
+                                                                </li>                        
+                                </div>
+    
+        
+                           
+                        </div>
+                </ul>
+            
+
+        
+
+        <div class="page1__body">
+            <h1 class="page1__heading">
+                Experts are here solve
+                your business problem.
+            </h1>
+            <p class="page1__para">
+                We know how large objects will act, but things on a
+                small scale just do not act that way.
+            </p>
+            <div class="page1__btns-block">
+                <button class="page1__btn1">
+                    <p class="page1__btn1-words">Get Quote Now</p>
+                </button>
+                <button class="page1__btn2">
+                    <p class="page1__btn2-words">Learn More</p>
+                </button>
+            </div>
+        </div>
+    </div>
+
+
+
+
+    <div class="page2">
```

Comment:
Why page2, instead reuse page1 block with modifiers, and try to use descriptive names.